src/quo/components/profile/collectible_list_item/style.cljs

```diff
@@ -91,8 +105,20 @@
    :background-color (colors/theme-colors colors/white-70-blur colors/neutral-95-opa-70-blur theme)
    :z-index          2})
 
+(defn loading-image-with-opacity
+  [theme opacity]
+  [{:opacity opacity}
+   (loading-image theme)])
+
 (defn avatar-container
-  [loaded?]
-  {:flex           1
-   :flex-direction :row
-   :opacity        (when-not loaded? 0)})
+  [opacity]
+  [{:opacity opacity}
+   {:flex           1
+    :flex-direction :row
+    :opacity        default-opacity-for-image}])
+
+(defn supported-file
+  [opacity]
+  [{:opacity opacity}
+   {:aspect-ratio 1
+    :opacity      default-opacity-for-image}])
```

Comment:
Good to see the vector syntax is being used :+1:  it's great because when it's possible to be used, it removed the complexities of dealing with a hook :+1: